pkg/gen/input/workflows/internal/file/create_release.go

```diff
@@ -189,10 +198,111 @@ jobs:
         run: |
           git push "${REMOTE_REPO}" --tags
       - name: Create release
+        id: create_gh_release
         uses: actions/create-release@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           tag_name: "v${{ needs.gather_facts.outputs.version }}"
           release_name: "v${{ needs.gather_facts.outputs.version }}"
+
+##-ignore-left-##- if eq .CurrentFlavour .FlavourCLI##-ignore-right-##
```

Comment:
When we use delimiter let's use something more sane. Maybe `{{{{` or something. The original problem was that GH actions use `${{ ... }}` and Go templates think `{{ ... }}` is it's delimiter.